packages/driver/src/cy/commands/waiting.ts

```diff
@@ -62,6 +63,16 @@ export default (Commands, Cypress, cy, state) => {
         // avoid circular reference
         options: _.omit(options, '_log'),
       })
+
+      // if this came from the spec bridge, we need to set a few additional
+      // properties to ensure the log displays correctly
+      if (options.isCrossOriginSpecBridge) {
+        log.set({
+          displayName: 'wait',
+          name: 'wait',
+          message: '',
```

Comment:
Doesn't this hide the wait arguments?

Reply:
No, the aliases are still displayed:
<img width="581" alt="Screen Shot 2022-05-20 at 11 55 56 AM" src="https://user-images.githubusercontent.com/2002044/169594182-d54c193d-6ff3-4e57-99b6-d809d665c64a.png">

Reply:
Clearing the message is also needed otherwise it will display details from the command 'origin' command which is running on the primary.